AirportDictionary/AirportDictionary/Pages/Index.cshtml.cs

```diff
@@ -20,47 +20,42 @@ public IndexModel(ILogger<IndexModel> logger)
             _logger = logger;
         }
 
-        public void OnGet()
+        public JsonResult OnGet()
```

Comment:
I urge caution here: you're changing functionality, which is not the intention of a code review.  This codebehind currently renders an HTML page, which may be what the group wants.  You're changing this to return JSON.